docs/build-instructions/windows.md

```diff
@@ -34,6 +34,7 @@ To also install the newly built application, use `script\build --create-windows-
 * `--code-sign`: signs the application with the GitHub certificate specified in `$WIN_P12KEY_URL`.
 * `--compress-artifacts`: zips the generated application as `out/atom-windows.zip` (requires 7-zip).
 * `--create-windows-installer`: creates an `.msi`, an `.exe` and a `.nupkg` installer in the `out/` directory.
+* `--install`: installs the application in `C:\Users\<user name>\AppData\Local\`.
```

Comment:
Slight nitpick: It would be clearer I think if this was `%LOCALAPPDATA%\Atom\app-dev\`.  Or if there's a reason the trailing Atom\\app-dev isn't included, then at least `%LOCALAPPDATA%`.